### PR TITLE
feat: core changes for application style alignment

### PR DESCRIPTION
**Pull request by emilyjablonski**
# Pull Request Template

## Issue Overview

[This PR addresses #issue](https://github.com/metrotranscom/doorway/issues/364)

- [x] This change addresses only certain aspects of the issue

## Description

From the linked Figma, this PR adds the styles that were indicated as also being core styles, and uptakes seeds in some places in the application to support overriding when we pull over to Doorway.

* Uptake of seeds Card component in the application
* Removing placeholder text from fields
* Removing caps styling from fields
* Standardizing text placement and header styling across different questions

I created a share-able layout for the structure of an application page as that was duplicated across all pages.

## How Can This Be Tested/Reviewed?

Compare the application on this branch to the Figma and/or to dev, there's lots of great documentation in there about what is a core change but there were lots of small changes so feel free to reach out if you're not sure if something is intentional!

Ensure that each page has the right new styles and that you can navigate forward / backward as in dev - [this doc](https://docs.google.com/document/d/1-ENOXNxGsYLI4TsCvKeO7nGz6FCiQsCrkjcxW19k6jU/edit?usp=sharing) has screenshots of each page compared to dev with notes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: sites/public/src/pages/applications/start/autofill.tsx [hunk 1]
```diff
@@ -97,23 +91,19 @@ export default () => {
...
-          <h2 className="form-card__title is-borderless mt-4">
-            {t("application.autofill.saveTime")}
-          </h2>
-        </div>
-        <div className="form-card__pager-row px-16">
-          <p className="field-note py-2">{t("application.autofill.prefillYourApplication")}</p>
-        </div>
+      <ApplicationFormLayout
+        listingName={listing?.name}
+        heading={"Save time by using the details from your last application."}
```
Comment (emilyjablonski) [inline sites/public/src/pages/applications/start/autofill.tsx:96]: todo

File: sites/public/src/pages/applications/start/choose-language.tsx [hunk 2]
```diff
@@ -108,63 +106,57 @@ const ApplicationChooseLanguage = () => {
...
-            {t("application.chooseLanguage.letsGetStarted")}
-          </h1>
-        </div>
-
+      <ApplicationFormLayout
+        listingName={listing?.name}
+        heading={t("application.chooseLanguage.letsGetStarted")}
+        progressNavProps={{
+          currentPageSection: 1,
+          completedSections: 0,
```
Comment (emilyjablonski) [inline sites/public/src/pages/applications/start/choose-language.tsx:114]: bug fix

File: shared-helpers/src/views/multiselectQuestions.tsx [hunk 3]
```diff
@@ -261,7 +261,7 @@ export const getCheckboxOption = (
       {watchFields[optionFieldName] && option.collectAddress && (
         <div className="pb-4">
           <FormAddress
-            subtitle={t("application.preferences.options.address")}
+            subtitle={""}
```
Comment (emilyjablonski) [inline shared-helpers/src/views/multiselectQuestions.tsx:264]: This can be removed w the next UIC PR

File: sites/public/src/components/applications/ApplicationMultiselectQuestionStep.tsx [hunk 4]
```diff
@@ -150,83 +143,59 @@ const ApplicationMultiselectQuestionStep = ({
...
-                  })}
+                  <legend className="text__caps-spaced mb-4 sr-only">{question?.text}</legend>
+                  {applicationSection === ApplicationSection.preferences && (
+                    <p className="field-note mb-8">{question?.description}</p>
+                  )}
+                  <p className="field-note mb-8">
+                    {t("application.household.preferredUnit.optionsLabel")}
+                  </p>
+                  {allOptions
+                    ?.sort((a, b) => (a.ordinal > b.ordinal ? 1 : -1))
```
Comment (emilyjablonski) [inline sites/public/src/components/applications/ApplicationMultiselectQuestionStep.tsx:187]: We weren't showing options by ordinal

**Comment by jaredcwhite** [comment]
I think we'll get the public cypress tests working once this and the button PR merge, so I'm going to punt on that for the moment.

File: shared-helpers/src/views/multiselectQuestions.tsx [hunk 5]
```diff
@@ -261,7 +261,6 @@ export const getCheckboxOption = (
       {watchFields[optionFieldName] && option.collectAddress && (
         <div className="pb-4">
           <FormAddress
-            subtitle={t("application.preferences.options.address")}
```
Comment (jaredcwhite) [inline shared-helpers/src/views/multiselectQuestions.tsx:264]: OK, so this is weird. This subtitle prop was removed in the PR, but recently changed on main. Should we keep it removed, or add it back in??

Comment (emilyjablonski) [reply shared-helpers/src/views/multiselectQuestions.tsx:264]: Given the Figma changes I think we're good to keep it removed

Comment (emilyjablonski) [reply shared-helpers/src/views/multiselectQuestions.tsx]: We're good, this is resolved from the last UIC change

**Comment by emilyjablonski** [comment]
To help visual QA, [here's a doc](https://docs.google.com/document/d/1-ENOXNxGsYLI4TsCvKeO7nGz6FCiQsCrkjcxW19k6jU/edit?usp=sharing) with screenshots of dev and this branch on every application page

File: shared-helpers/src/locales/zh.json [hunk 6]
```diff
@@ -144,22 +144,22 @@
...
   "application.household.member.cancelAddingThisPerson": "取消加入此人",
   "application.household.member.dateOfBirth": "出生日期",
   "application.household.member.deleteThisPerson": "刪除此人",
   "application.household.member.haveSameAddress": "他們的地址是否和您一樣？",
   "application.household.member.name": "家庭成員姓名",
   "application.household.member.saveHouseholdMember": "儲存家庭成員資料",
-  "application.household.member.subTitle": "在下一個頁面，您將有機會加入更多家庭成員",
-  "application.household.member.title": "請提供此人的資料",
+  "application.household.member.subTitle": "在下一個頁面，您將有機會加入更多家庭成員。",
+  "application.household.member.title": "請提供此人的資料。",
```
[the remark this answers is not in the document]
Comment (emilyjablonski) [reply shared-helpers/src/locales/zh.json:163]: Yep! This was a change from adding periods at the end of some of the titles.

File: sites/public/src/components/shared/FormSummaryDetails.tsx [hunk 7]
```diff
@@ -470,7 +474,12 @@ const FormSummaryDetails = ({
               label={t("t.income")}
               className={"pb-4"}
             >
-              ${application.income} {t(`t.${application.incomePeriod}`)}
+              $
```
Comment (emilyjablonski) [inline sites/public/src/components/shared/FormSummaryDetails.tsx:477]: This is an unrelated bug fix - we weren't showing commas or decimals correctly when displaying the income value

**Review by ludtkemorgan — APPROVED** [review]
Thanks for making those changes! Looks like some tests are failing, but otherwise looks good!
